package.json

```diff
@@ -62,8 +62,8 @@
     "react-dom-factories": "^1.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
```

Comment:
Are you removing compatibility for `^0.14.0 || ^15.0.0`?

Reply:
Not really, it's just a initial PR :)

Reply:
I'm forcing this temporary just to test what would need to change in regarding to support react 16. If possible, I'll guarantee backwards compatibility with previous version of react.